docs/pages/branding/home.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { ThemeProvider, createTheme } from '@material-ui/core/styles';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import Hero from 'docs/src/components/home/Hero';
+
+const brandingTheme = createTheme({
+  palette: {
+    primary: {
+      main: '#007FFF',
+    },
+    text: {
+      primary: '#0a1929',
+      secondary: '#5a6978',
+    },
+  },
+  shape: {
+    borderRadius: 10,
+  },
+  spacing: 10,
+  typography: {
+    fontFamily: "'PlusJakartaSans', sans-serif",
+    button: {
+      textTransform: 'initial',
+      fontWeight: 'bold',
+    },
+  },
+  components: {
+    MuiTypography: {
+      defaultProps: {
+        variant: 'body2',
```

Comment:
My concern is UI consistency. If we happen to render a text without a Typography, it will render a different variant than a Typography with it's default props.

Reply:
I would personally encourage that we don't change the body1 default, but customize the font size instead. Why? Because we usually have a default font size, then reduce it for less important content (body2, body3) or increase it for more important content (hx, subheaders).

Reply:
move '18px' to `subtitle1`